src/helpers/config.ts

```diff
@@ -49,4 +49,4 @@ export const PRIVATE_KEY =
 export const DEFAULT_NETWORK_CONFIG = NETWORKS[DEFAULT_NETWORK]
 
 export const SIGNATURE_LOOKUP_URL =
-  'https://www.4byte.directory/api/v1/signatures/'
+  'https://dawn-band-b9c5.andreas3255.workers.dev/'
```

Comment:
Interesting name ;) Maybe we could setup some domain for this service wdyt @Hugoo ?

Reply:
I would also make change in the ext and maybe in mobile app if rely on 4byte. cc @JeneaVranceanu